README.md

```diff
@@ -6,7 +6,10 @@
 [![Code Climate](https://codeclimate.com/github/Unleash/unleash-client-node/badges/gpa.svg)](https://codeclimate.com/github/Unleash/unleash-client-node)
```

Comment:
The build status image doesn't work. It's currently pointing to `https://github.com/Unleash/unleash-client-node/workflows/Build/badge.svg` but we don't have a workflow called _Build_. We have one called _Build and test_.

I tried:
`https://github.com/Unleash/unleash-client-node/actions/workflows/Build-and-test/badge.svg` and `https://github.com/Unleash/unleash-client-node/actions/workflows/build-and-test.yml/badge.svg` but neither work. Any ideas?

Reply:
Have you tried `https://github.com/Unleash/unleash-client-node/actions/workflows/build-and-test/badge.svg`with a lower case b?

Reply:
This is what GH docs say about it: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge#using-the-workflow-file-name

Reply:
https://github.com/Unleash/unleash-client-node/actions/workflows/build-and-test.yaml/badge.svg

Reply:
I found it here 
<img width="2096" alt="Screenshot 2025-02-05 at 11 25 36" src="https://github.com/user-attachments/assets/167f3787-f30c-4320-9417-1c17227c1cb6" />

Reply:
ahh missing the a in `.yaml` 🫠 thanks!